Configure.pl

```diff
@@ -555,16 +555,15 @@ sub uniq {
                         . "\t\$(CP) 3rdparty/msinttypes/*.h \"\$(DESTDIR)\$(PREFIX)/include/msinttypes\"\n";
 }
 
+if ($^O eq 'aix' && $config{ld} =~ /xlc/ && $config{ptr_size} == 4) {
```

Comment:
It really won't matter but from a very quick research, $config{ld} can be a path and the linker command can be like xlc, xlc_v6, xlc128 or the like, so I'd feel more comfortable with a tightened regex like `m!/xlc`

Reply:
I'm reworking this branch rebased onto #1417 and given that AIX needs fixing "properly", I'm going to remove this regex from this (rebased) commit to keep things simple.

"properly" means that we need to handle different toolchains on the same OS, which is what #1417 starts to enable.